dtd-parser/src/main/java/com/sun/xml/dtdparser/DTDParser.java

```diff
@@ -330,7 +329,6 @@ private void parseInternal(InputSource input)
                 fatal("P-003", null);
             }
         } catch (RuntimeException e) {
-            LOGGER.log(Level.SEVERE, "Internal DTD parser error.", e);
```

Comment:
Why are you removing logger? It should stay there

Reply:
My mistake. I thought that it was unuseful. Restored in https://github.com/eclipse-ee4j/jaxb-dtd-parser/pull/41/commits/2ac4c77fa9702c42fc217fd0444fd1c1b64f422d